File: Python3/prestacao.py

"""Problema: Analisando a fórmula " Prestação = valor + (valor * (taxa/100) * tempo)",
crie um algoritmo para efetuar o cálculo do valor de uma prestação em atraso.
 (Você deverá ler o VALOR da prestação, a TAXA  de juros imposta pelo banco,
 e o número de dias em ATRASO"""


valor_prest = float(input("Digite o valor da prestacao.: "))
taxa = float(input("Digite a taxa de juros: "))
dias_atraso = int(input("\ndigite o numero de dias em atraso: "))
prestacao = valor_prest + (valor_prest * (taxa/100)*dias_atraso)
print("O valor corrigido da prestacao = %f"%(prestacao))

File: java/prestacao.java
import java.util.Scanner;

class prestacao {
   public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      float prestacao, valor_prest, taxa;
      int dias_atraso;

      System.out.println("Digite o valor da prestacao");
      valor_prest = sc.nextFloat();
      System.out.println("Digite a taxa de juros");
      taxa = sc.nextFloat();
      System.out.println("digite o numero de dias em atraso");
      dias_atraso = sc.nextInt();

      prestacao = valor_prest + (valor_prest * (taxa / 100) * dias_atraso);

      System.out.println("O valor corrigido da prestacao =  " + prestacao);
      sc.close();
   }
}
